applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/prebuckling_strategy.hpp

```diff
@@ -690,6 +748,36 @@ class PrebucklingStrategy
     ///@name Protected Operations
     ///@{
 
+    /**
+     * @brief This method assigns settings to member variables
+     * @param ThisParameters Parameters that are assigned to the member variables
+     */
+    void AssignSettings(const Parameters ThisParameters) override
+    {
+        BaseType::AssignSettings(ThisParameters);
+        mMaxIteration = ThisParameters["max_iteration"].GetInt();
+        mInitialLoadIncrement = ThisParameters["buckling_settings"]["initial_load_increment"].GetDouble();
+        mSmallLoadIncrement = ThisParameters["buckling_settings"]["small_load_increment"].GetDouble();
+        mPathFollowingStep = ThisParameters["buckling_settings"]["path_following_step"].GetDouble();
+        mConvergenceRatio = ThisParameters["buckling_settings"]["convergence_ratio"].GetDouble();
+        mMakeMatricesSymmetricFlag = ThisParameters["buckling_settings"]["make_matrices_symmetric"].GetBool();
+
+        // Saving the convergence criteria to be used
+        if (ThisParameters["convergence_criteria_settings"].Has("name")) {
+            KRATOS_ERROR << "IMPLEMENTATION PENDING IN CONSTRUCTOR WITH PARAMETERS" << std::endl;
```

Comment:
pending implementation is a blocker...

Reply:
I will have a look, I did this 2 years ago, and I don't remember details

Reply:
This is pending because we don't have the CC created with factories yet, it is not possible to add before factories or the new proposal is added. Remember this was originally planned as a transition change!